.travis.yml

```diff
@@ -27,19 +27,19 @@ stages:
   
 jobs:
   include:
-    # - name: "WhiteSource scan"
-    #   stage: build
-    #   # only on SP branches or master and if it is not a PR
-    #   if: fork = false AND (branch = develop OR branch =~ /support\/SP\/.*/ OR branch =~ /^APPS-.*$/) AND type != pull_request
-    #   script:
-    #     # Download the latest version of WhiteSource Unified Agent
-    #     - curl -LJO https://github.com/whitesource/unified-agent-distribution/releases/latest/download/wss-unified-agent.jar
-    #     # Run WhiteSource Unified Agent
-    #     - java -jar wss-unified-agent.jar -apiKey ${WHITESOURCE_API_KEY} -c _ci/.wss-unified-agent.config 
+#    - name: "WhiteSource scan"
+#      stage: build
+#      # only on SP branches or master and if it is not a PR
+#      if: fork = false AND (branch = develop OR branch =~ /support\/SP\/.*/ OR branch =~ /^APPS-.*$/) AND type != pull_request
+#      script:
+#        # Download the latest version of WhiteSource Unified Agent
+#        - curl -LJO https://github.com/whitesource/unified-agent-distribution/releases/latest/download/wss-unified-agent.jar
+#        # Run WhiteSource Unified Agent
+#        - java -jar wss-unified-agent.jar -apiKey ${WHITESOURCE_API_KEY} -c _ci/.wss-unified-agent.config
     - name: "Source Clear Scan (SCA)"
       stage: build
       # only on SP branches or master and if it is not a PR
-      if: fork = false AND (branch = develop OR branch =~ /support\/SP\/.*/ OR branch =~ /^APPS-.*$/) AND type != pull_request
+      if: fork = false AND (branch =~ /support\/.*/) AND type != pull_request AND commit_message !~ /\[no-release\]/
```

Comment:
so, we execute sourceclear agent just:
1-when is not a fork
2-when branch is support/something
3-when type is not pull_request
4-when commit message doesn't contain no-release
On point 4 i think we dont need it. We should execute source clear always, you can remove the last condition